spec/helpers/account_price_group_members_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe AccountPriceGroupMembersHelper do
+  describe "additional_results_notice" do
+    it "returns nil if an input is missing" do
+      expect(additional_results_notice(count: nil, limit: 25)).to be_nil
+      expect(additional_results_notice(count: 14, limit: nil)).to be_nil
+      expect(additional_results_notice(count: 14, limit: 25)).to be_nil
+    end
+
+    it "returns nil if the accounts_count isn't over the limit" do
+      expect(additional_results_notice(count: 12, limit: 25)).to be_nil
+	end
+
+    it "returns a notice if accounts_count is over the limit" do
```

Comment:
Looks like some indenting issues here.  Can you set 2 spaces as the default in VSCode?